<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MobilityRecon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_window.qrc">
    <normaloff>:/MainWindow/Resources/Cute-Ball-Favorites-icon.png</normaloff>:/MainWindow/Resources/Cute-Ball-Favorites-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport"/>
    <addaction name="actionSnapshot"/>
    <addaction name="actionExportSequentialSnapshots"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionScanByKinect2"/>
    <addaction name="actionStopScan"/>
    <addaction name="actionComputeNormalsForFrames"/>
    <addaction name="separator"/>
    <addaction name="actionSaveWhenScanning"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionImport"/>
   <addaction name="actionSnapshot"/>
   <addaction name="actionExportSequentialSnapshots"/>
   <addaction name="separator"/>
   <addaction name="actionScanByKinect2"/>
   <addaction name="actionStopScan"/>
   <addaction name="actionComputeNormalsForFrames"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/fileopen.png</normaloff>:/MainWindow/Resources/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/Crystal_Project_Exit.png</normaloff>:/MainWindow/Resources/Crystal_Project_Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/fileimport.png</normaloff>:/MainWindow/Resources/fileimport.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/snap_shot.png</normaloff>:/MainWindow/Resources/snap_shot.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
  <action name="actionExportSequentialSnapshots">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/export_sequential_snapshots.png</normaloff>:/MainWindow/Resources/export_sequential_snapshots.png</iconset>
   </property>
   <property name="text">
    <string>Export Sequential Snapshots</string>
   </property>
  </action>
  <action name="actionScanByKinect2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/kinect.png</normaloff>:/MainWindow/Resources/kinect.png</iconset>
   </property>
   <property name="text">
    <string>Scan By Kinect2</string>
   </property>
  </action>
  <action name="actionStopScan">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/stopscan.png</normaloff>:/MainWindow/Resources/stopscan.png</iconset>
   </property>
   <property name="text">
    <string>Stop Scan</string>
   </property>
  </action>
  <action name="actionSaveWhenScanning">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save When Scanning</string>
   </property>
  </action>
  <action name="actionComputeNormalsForFrames">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/MainWindow/Resources/normal.png</normaloff>:/MainWindow/Resources/normal.png</iconset>
   </property>
   <property name="text">
    <string>Compute Normals For Frames </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="main_window.qrc"/>
 </resources>
 <connections/>
</ui>
